Domain/Services/Abstract/IProjectService.cs

```diff
@@ -0,0 +1,11 @@
+using Domain.DTOs;
+using Domain.Entities;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Domain.Services.Abstract
+{
+    public interface IProjectService : IBaseService<ProjectDTO>
```

Comment:
If we remove `IBaseService.cs` , we can update this 
```suggestion
    public interface IProjectService 
```